views/sign-out.html

```diff
@@ -16,12 +16,10 @@
   }) }}
 {% endblock %}
 
-{% if saveAndResume and (journey == "register") %}
+{% if journey == "register" %}
   {% set hintText = "We will save your application." %}
-{% elif saveAndResume and (journey == "update") %}
+{% elif journey == "update" %}
```

Comment:
looks like the controllers/update/update.sign.out.controller.ts will still need to pass a saveAndResume value into this template. So i think we would need to revert this template and change the controllers/sign.out.controller.ts to pass saveAndResume: true into this template, until the update flags are removed.

Reply:
Back out this change as update save and resume is still being used.